<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendMoneyFormClass</class>
 <widget class="QWidget" name="SendMoneyFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SendMoneyForm</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Send Money</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Recipient's Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUsername">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    padding: 6px 10px;
    font-size: 14px;
    background: transparent;

}

QLineEdit:focus {
    border: 1px solid #7f8cff;
}</string>
   </property>
   <property name="placeholderText">
    <string>Enter Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxAmount">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    padding: 6px 10px;
    font-size: 14px;
    background: transparent;

}

QDoubleSpinBox:focus {
    border: 1px solid #7f8cff;
}</string>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::PlusMinus</enum>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>10.000000000000000</double>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Note (Optional)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNote">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    padding: 6px 10px;
    font-size: 14px;
    background: transparent;

}

QLineEdit:focus {
    border: 1px solid #7f8cff;
}</string>
   </property>
   <property name="placeholderText">
    <string>What's this for?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSend">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>420</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 10px;
    font-size: 14px;
}

QPushButton:enabled {
    background-color: #6C63FF;
    color: white;
    border: none;
}

QPushButton:hover {
    background-color: #5a55d3;
}

QPushButton:pressed {
    background-color: #4a46b2;
}</string>
   </property>
   <property name="text">
    <string>Send Money</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #f1f1f5;
    color: #000000;
    border: none;
    border-radius: 10px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #b1b1b1;
}

QPushButton:pressed {
    background-color: #a1a1a1;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
